coral-component-colorinput/src/scripts/ColorInputSwatch.js

```diff
@@ -160,7 +167,7 @@ class ColorInputSwatch extends BaseColorInputAbstractSubview(BaseComponent(HTMLE
   static get observedAttributes() {
     return super.observedAttributes.concat([
       'selected',
-      'tabindex',
+      'tabIndex',
```

Comment:
@majornista  Do you notice any difference using `tabIndex` instead of `tabindex` ? AFAIR it should be equivalent because attributes are not case sensitive.

Reply:
I don't really notice a difference. Not sure why I changed the case here; the attributes are usually lowercase, while the properties use camelCase.

Reply:
I think I may have been thinking in React.